app-foundation/3-networks/development/firewall.tf

```diff
@@ -0,0 +1,311 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+ VPC firewall rules
+*****************************************/
+
+locals {
+  boa_gke_cluster1_master_cidr = "100.64.78.0/28"
+  boa_gke_cluster2_master_cidr = "100.65.70.0/28"
+  boa_gke_mci_master_cidr      = "100.64.70.0/28"
+}
+
+resource "google_compute_firewall" "allow_lb_healthcheck" {
```

Comment:
lb health check FW rule is in foundations, can you check if that satisfies this?

https://github.com/terraform-google-modules/terraform-example-foundation/blob/0d315491bc3ad7b9578b40042ec97907c4b69cd5/3-networks/modules/base_shared_vpc/firewall.tf#L162

Reply:
This should work - I'll remove the repeated firewall rule in my code.